build/yaml/javascriptDeploySteps.yml

```diff
@@ -54,7 +54,7 @@ steps:
     inlineScript: |
      Set-PSDebug -Trace 1;
      # set up resource group, bot channels registration, app service, app service plan
-     az deployment sub create --name "$(BotName)$(Build.BuildId)" --template-file "$(TemplateLocation)" --location "westus" --parameters appId="$(DeployAppId)" appSecret="$(DeployAppSecret)" botId="$(BotName)" botSku=F0 newAppServicePlanName="$(BotName)" newWebAppName="$(BotName)-$(Build.BuildId)" groupName="$(BotGroup)" groupLocation="westus" newAppServicePlanLocation="westus"
+     az deployment sub create --name "$(BotName)$(Build.BuildId)" --template-file "build\templates\template-bot-resources-legacy.json" --location "westus" --parameters appId="$(DeployAppId)" appSecret="$(DeployAppSecret)" botId="$(BotName)" botSku=F0 newAppServicePlanName="$(BotName)" newWebAppName="$(BotName)-$(Build.BuildId)" groupName="$(BotGroup)" groupLocation="westus" newAppServicePlanLocation="westus"
```

Comment:
I think @mrivera-ms recently update main to default to S1 (see https://github.com/microsoft/BotFramework-FunctionalTests/pull/241/files), do we need that here too? or we override this in the build parameters?